packages/hooks/src/useP2POrderList.ts

```diff
@@ -0,0 +1,15 @@
+import { useFetch } from '@deriv/api';
+import { isMobile } from '@deriv/shared';
+
+/** A custom hook to get the list of p2p order list. */
+const useP2POrderList = () => {
+    const list_item_limit = isMobile() ? 10 : 50;
+    const { data, ...rest } = useFetch('p2p_order_list', { payload: { active: 0, offset: 0, limit: list_item_limit } });
```

Comment:
As @farzin-deriv mentioned let's use usePaginatedFetch instead of this :thinking: